molecule/scenario_with_local_binary_directory/molecule.yml

```diff
@@ -0,0 +1,28 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint: |
+  set -e
+  yamllint .
+  flake8
+
+platforms:
+  - name: instance
+    image: quay.io/centos/centos:stream8
+    command: /sbin/init
+    privileged: True
```

Comment:
Can you elaborate why the change to a different image is required, compared to the default setup of the role in i.e.
https://github.com/patrickjahns/ansible-role-promtail/blob/master/molecule/default/molecule.yml#L12-L19 ?

Reply:
When using this image : 

```
  - name: instance
    image: "geerlingguy/docker-${MOLECULE_DISTRO:-ubuntu2204}-ansible:latest"
    command: ${MOLECULE_DOCKER_COMMAND:-""}
    volumes:
      - /sys/fs/cgroup:/sys/fs/cgroup:rw
    privileged: true
    cgroupns_mode: host
    pre_build_image: true
```
I encounter an issue with the test which is checking if the promtail service is running.

Maybe, it is because by systemd integration on docker and because in run molecule on a virtualbox VM.
It was my only way to found a solution to this issues but I can put the same image for the CI integration

Reply:
🤔  - strange - do you still have the issue at hand ? 
A while ago I had to solve some issues with systemd and the containers as well ( https://github.com/patrickjahns/ansible-role-promtail/pull/133/files#diff-90c2024f9375b6ff2916ee376a3be4f93728b19faae7a13da75e89719472390cR180 ). Maybe it's the same issue that you encountered

Reply:
Ok, I have try to use molecule without on my host directly and not on a virtualbox VM. I confirm that it is a issue I encounter only on docker running on virtualbox.

I will push changes to use geerlingguy image